---
title: "Research"
bg: green1
color: black
fa-icon: cogs
---
<div class="container"> 
<p>
My areas of interest broadly cover probabilistic graphical models, computational statistics, and causal inference. I mainly work on developing methods using principles from these areas for analysing large scale perturbation screens data. Here you can find a list of my relevant publications. </p> 

<ul>
  <li style="font-size:15px"> <strong>S. Srivatsa</strong><sup style="font-size: 70%;vertical-align: super;">&#10018;</sup>, H. Montazeri<sup style="font-size: 70%;vertical-align: super;">&#10018;</sup>, G. Bianco, M. Coto-Llerena, S. Piscuoglio, CKY. Ng, N. Beerenwinkel (2019). <a href="https://www.biorxiv.org/content/10.1101/810374v1" style="color: #2E2D2D;text-decoration: underline;"> Discovery of synthetic lethal interactions from large-scale pan-cancer perturbation screens </a>, <em>In review</em>, Preprint bioRxiv: 10.1101/810374. </li>
  <li style="font-size:15px"> H. Montazeri, M. Coto-Llerena, G. Bianco, E. Zangeneh,  S. Taha-Mehlitz, V. Paradiso, <strong>S. Srivatsa</strong>,  A. de Weck,  G. Roma,  M. Lanzafame, M. Bolli,  N. Beerenwinkel, M. von Flue,  LM. Terracciano, S. Piscuoglio, CKY. Ng (2019). <a href="https://www.biorxiv.org/content/10.1101/807248v1" style="color: #2E2D2D;text-decoration: underline;"> APSiC: Analysis of Perturbation Screens for the Identification of Novel Cancer Genes</a>,  <em>In review</em>, Preprint bioRxiv: 10.1101/807248. </li>
  <li style="font-size:15px"> T. Ringel, N. Frey, F. Ringnalda, S. Janjuha, S. Butz, S. Cherkaoui, <strong>S. Srivatsa</strong>, M. Pirkl, G. Russo, G. Rogler, L. Villiger, N. Beerenwinkel, N. Zamboni, T. Baubec,  G. Schwank (2019). <a href="https://www.sciencedirect.com/science/article/abs/pii/S1934590920300606" style="color: #2E2D2D;text-decoration: underline;"> Genome-scale CRISPR screening in human intestinal organoids identifies drivers of TGF-&beta; resistance</a>,  <em> Cell Stem Cell</em>, doi: 10.1016/j.stem.2020.02.007.</li>
  <li style="font-size:15px"> <strong>S. Srivatsa </strong>, J. Kuipers, F. Schmich, S. Eicher, M. Emmenlauer, C. Dehio, N. Beerenwinkel (2018). <a href="https://academic.oup.com/bioinformatics/article/34/13/i519/5045779" style="color: #2E2D2D;text-decoration: underline;"> Improved pathway reconstruction from RNA interference screens by exploiting off-target effects</a>, <em>Bioinformatics</em>, doi: 10.1093/bioinformatics/bty240. </li>
  <li style="font-size:15px"> C. Wang, F. Schmich, <strong>S. Srivatsa</strong>, J. Weidner, N. Beerenwinkel, A. Spang (2018). <a href="https://elifesciences.org/articles/29815" style="color: #2E2D2D;text-decoration: underline;"> Context-dependent deposition and regulation of mRNAs in P-bodies</a>,  <em> eLife</em>, doi: 10.7554/eLife.29815.</li>
</ul>

<p style="font-size:15px"> 
&#10018; Authors contributed equally </p>
</div>


<!-- 
#### Pathway reconstruction exploting off-target effects

<div class="container">
  <div class="column halfx" align="left">
	Pathway reconstruction has proven to be an indispensable tool for analyzing the underlying molecular mechanisms of a cell. There are existing probabilistic graphical models, which have been designed to reconstruct pathways from high dimensional observations resulting from RNA interference (RNAi) experiments. These models assume that the short-interfering RNAs (siRNAs) designed to knockdown specific genes are always on-target. However, in reality most siRNAs exhibit strong off-target effects, which further confound the RNAi screen data, thus resulting in unreliable reconstruction of networks. We developed a method called probabilistic combinatorial nested effects model (pc-nem) to use this combinatorial knockdown data as a result of siRNA off-target effects to accurately infer the underlying pathway. <br/><br> 
	For more information, check out our paper in <a href="https://academic.oup.com/bioinformatics/article/34/13/i519/5045779" style="color: #2E2D2D;text-decoration: underline;">Bioinformatics</a> as a part of ISMB 2018 proceedings. The software is freely available as an R package on <a href="https://github.com/cbg-ethz/pcNEM.git" style="color: #2E2D2D;text-decoration: underline;">CBG's github</a>. <br/><br/><br/>
  </div>
  <div class="column halfx" align="center">
	<img align = "center" src = "img/pcnem.png" width="350">
  </div>
  
</div>

#### Discovering mutation-specific synthetic lethals

<div class="container">
  <div class="column halfx" align="center">
	<img align = "center" src = "img/slidr.png" width="400">
  </div>
  <div class="column halfx" align="left">
	The concept of synthetic lethality (SL) exploits the underlying genetic interactions in cancer and holds the promise of novel therapeutic target identification.  However, identifying such mutation-specific SL targets still remains an active challenge in oncology. We developed SLIdR (Synthetic Lethal Identification in R), a novel method for mining mutation-specific SL partners exclusively from large-scale perturbation screen data. SLIdR successfully predicts SL pairs even with small sample sizes and controls the false positive rate. We applied SLIdR across 20 different cancer types and identified novel as well as established pan-cancer and cancer-specific SL pairs, in good agreement with knowledge from literature. Further, SLIdR identified a novel SL interaction between AXIN1 and URI1, which was comprehensively validated in hepatocellular carcinoma cell lines.<br>
	<br>
  	For more information, check out <a href="https://docs.google.com/document/d/15S1vvHTrKEx8qbNAF5BdrhFPn30STXlVeACKNI0BzKQ/edit?usp=sharing" style="color: #2E2D2D;text-decoration: underline;">our paper</a>. The software is freely available as an R package on <a href="https://github.com/cbg-ethz/slidr/" style="color: #2E2D2D;text-decoration: underline;">CBG's github.</a>. 
  </div>
  
</div>
-->




